Update how to handle Aurora cluster with Docker (#244)
pganalyze/pganalyze-docs: install/amazon_rds/04_configure_the_collector_docker.mdx

## Handling Amazon Aurora clusters vs instances

In the case of Amazon Aurora, the collector automatically resolves `cluster` endpoints to the underlying writer instance.

```
DB_HOST=mydbcluster.cluster-123456789012.us-east-1.rds.amazonaws.com
```

> This will only monitor the writer instance. If you also want to monitor a reader instance, you'll need to run another Docker container.
> Create a new configuration file (e.g. named `pganalyze_collector.replica.env`) and use the `cluster-ro` endpoint as the `DB_HOST`:

Reviewer: Technically, you could use CONFIG_CONTENTS to monitor the whole cluster with one collector. We do mention that in [our docs](https://pganalyze.com/docs/collector/settings) (separately, should it be clearer?). I feel like we should mention it here for customers who would rather just run a single collector. How about something like
```suggestion
This will only monitor the writer instance. If you also want to monitor a reader instance, you'll need to use the [`CONFIG_CONTENTS`](https://pganalyze.com/docs/collector/settings) configuration mechanism and add another section with the `cluster-ro` endpoint as the `db_host`.

Alternately, you can run a separate Docker container to monitor each node. Create a new configuration file (e.g. named `pganalyze_collector.replica.env`) and use the `cluster-ro` endpoint as the `DB_HOST`.
```

Author: Mentioning about `CONFIG_CONTENTS` here is a good idea, but I think that still can be a secondary option and we can suggest running two Docker containers as a primary option.
Or, do we actually want to push people to use `CONFIG_CONTENTS`?

Reviewer: Yeah, good point. I think running a single collector for multiple servers (especially when they're closely related like this) is more manageable, but the other way is a bit simpler. I'd lean to preferring `CONFIG_CONTENTS`, but no strong feelings.

```
DB_HOST=mydbcluster.cluster-ro-123456789012.us-east-1.rds.amazonaws.com
```

> If you have multiple readers you want to monitor, you need to specify each instance endpoint separately, running one pganalyze collector Docker container for each instance:

Reviewer: ```suggestion
If you have multiple readers you want to monitor, you will need to specify each instance endpoint separately:
```
Though I guess we should update the example that follows to use the INI file syntax and have two separate sections. We can elide most of the config except the two `db_host` lines and the section names. We could use section names of `[primary]` and `[replica]` to make it clearer, too.
